docs/welcome.mdx

```diff
@@ -4,18 +4,26 @@ title: Welcome to Ory!
 sidebar_label: Welcome
 ---
 
-Ory is an open source ecosystem and a cloud offering authentication, authorization, access control, and delegation (OAuth2 &
-OpenID Connect) services and APIs:
-
-- [Ory Cloud](https://console.ory.sh) is a cloud service built on top of Ory Open Source.
-- Ory Open Source offers scalable and hardened security software:
-- [Ory Kratos](https://github.com/ory/kratos)
-- [Ory Hydra](https://github.com/ory/hydra)
-- [Ory Oathkeeper](https://github.com/ory/oathkeeper)
-- [Ory Keto](https://github.com/ory/keto)
-
-Some parts of Ory are actively being developed. To learn more about roadmap and planned features head over to
-[the roadmap](./ecosystem/roadmap.md) and [project maturity](./ecosystem/versioning.md).
+Ory is a software infrastructure provider building a global zero trust network for humans, robots, devices, and logical services.
+Ory fuses open source software (Ory, Kubernetes, Crossplane, Cockroach, Linux, Istio) and open standards (Ory, OAuth2, OpenID
```

Comment:
I might not understand the intent here, but saying that Ory fuses Ory with Ory sounds weird. Maybe there's no need to add Ory to the lists in parentheses? I think it's self-explanatory.